core/types/transaction.go

```diff
@@ -101,6 +104,14 @@ type TxData interface {
 	effectiveGasPrice(dst *big.Int, baseFee *big.Int) *big.Int
 }
 
+func (tx *Transaction) PutOptions(options *OptionsAA4337) {
```

Comment:
same as above, comments to document the methods

Reply:
[Done](https://github.com/maticnetwork/bor/pull/945/commits/abe87af0bb99b7995b621af5eb41d50b4a964e0f).